docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>Would highlight that it works under jit as, e.g.: ...progress when tackling larger problems, and it also works under {func}<code>~jax.jit.</code></p><p> </p><p>I think it's time we exposed the <code>LRSinkhornState</code> in the docs, can you please add it to the appropriate place?</p><p>Would also link to <code>NamedTuple</code> as {class}<code>~typing.NamedTuple</code></p><p> </p><p>Slightly nicer:</p><p>... the same discussion also applies for the {class}<code>low-rank Sinkhorn solver &lt;ott.solvers.linear.sinkhorn_lr.LRSinkhorn&gt;</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I think `LRSinkhornState` still needs to be added to `docs/solvers/linear.rst` for the link to be generated.

Reply:
<p>Indeed, thanks.</p>